environ_dreamcast.sh

```diff
@@ -1,6 +1,16 @@
 # KallistiOS environment variable settings. These are the shared pieces
 # for the Dreamcast(tm) platform.
 
+# Add the default external DC tools path if it isn't already set.
+if [ -z "${DC_TOOLS_BASE}" ] ; then
+    export DC_TOOLS_BASE="/opt/toolchains/dc/bin"
```

Comment:
I'd rather we didn't put any absolute paths in here... Since this is a dc-specific thing as of now and this is the assumed default at this point, it should be fine to do it this way as a default.

```suggestion
    export DC_TOOLS_BASE="${KOS_CC_BASE}/../bin"
```